backends/tfhe-cuda-backend/cuda/src/device.cu

```diff
@@ -35,6 +40,12 @@ cudaStream_t cuda_create_stream(uint32_t gpu_index) {
   cuda_set_device(gpu_index);
   cudaStream_t stream;
   check_cuda_error(cudaStreamCreateWithFlags(&stream, cudaStreamNonBlocking));
+#ifndef NDEBUG
+  if (!g_debug_warning_printed)
```

Comment:
this is not very elegant, as it uses a global variable. 

with the global flag, running the `make test_integer_gpu_debug` target will only print the WARNING once

Reply:
Adding cargo build -vv  shows build logs